vaadin-dir-helper.html

```diff
@@ -0,0 +1,97 @@
+<!--
+@license
+Copyright (c) 2020 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<script>
+  window.Vaadin = window.Vaadin || {};
+
+  Vaadin.DirHelper = class VaadinDirHelper {
+    /**
+     * Get the scroll type in the current browser view.
+     *
+     * @return {String} the scroll type. Possible values are `default|reverse|negative`
+     */
+    static _detectScrollType() {
+      const inDOM = !!(typeof window !== 'undefined' && window.document && window.document.createElement);
```

Comment:
Seems like this check is for node.js so you can remove it completely.

Reply:
Removed.